src/platform_impl/macos/menu_item.rs

```diff
@@ -73,19 +74,36 @@ impl TextMenuItem {
     }
 
     pub fn label(&self) -> String {
-        todo!()
+        unsafe {
+            let title = msg_send![self.ns_menu_item, title];
+            std::str::from_utf8(CStr::from_ptr(title).to_bytes())
+                .unwrap()
+                .to_string()
```

Comment:
maybe using `*_lossy` is better to avoid a potential panic?
```suggestion
         std::str::from_utf8_lossy(CStr::from_ptr(title).to_bytes())
                .to_string()
```

Reply:
Thanks for the reminder, using `*_lossy` will be better.

I just found a proper way to get UTF8String and its length from NSString it self, I'll push another commit later.